log/logger.go

```diff
@@ -127,6 +128,9 @@ type Logger interface {
 	// SetHandler updates the logger to write records to the specified handler.
 	SetHandler(h Handler)
 
+	// SetSkipLevel sets the skip level of call stack
+	SetSkipLevel(skip int)
```

Comment:
what's the design concept here? do you want ppl to initiate a logger with default skip level and then set it at runtime? if so, why not just set it to the default in the struct and no need to pass it in as an argument.